src/sync.ts

```diff
@@ -100,13 +100,11 @@ const MSC2716_ROOM_VERSIONS = [
 function getFilterName(userId: string, suffix?: string): string {
     // scope this on the user ID because people may login on many accounts
     // and they all need to be stored!
-    return "FILTER_SYNC_" + userId + (suffix ? "_" + suffix : "");
+    return `FILTER_SYNC_${userId}${suffix ? "_" + suffix : ""}`;
```

Comment:
given the interpolated ternary on `suffix`, isn't the old version more readable? Can be split over lines:

```
return 'FILTER_SYNC_' +
  userId +
  suffix ? `_{suffix}` : ''
``` 
or